include/grpc/grpc_crl_provider.h

```diff
@@ -38,32 +38,26 @@ class Crl {
 
  protected:
   Crl() = default;
-
- private:
-  std::string raw_crl_;
 };
 
-// Opaque representation of a Certificate
-class Cert {
+// Information about a certificate to be used to fetch its associated CRL.
+class CertificateInfo {
  public:
-  static absl::StatusOr<std::unique_ptr<Cert>> Parse(
-      absl::string_view cert_string);
-  virtual ~Cert() = default;
+  explicit CertificateInfo(absl::string_view issuer) : issuer_(issuer) {}
+  absl::string_view GetIssuer() { return issuer_; }
 
  protected:
-  Cert() = default;
-
  private:
-  absl::string_view raw_cert_;
+  std::string issuer_;
```

Comment:
It's probably not necessary to store a copy of the string here, since this object will exist only during a single synchronous call to `GetCrl()`.  The gRPC code that constructs this `CertificateInfo` object will have its own copy of the cert info, so this can probably just contain a `string_view` -- or maybe it can even be a wrapper of some other object in the impl and can simply access the data directly from some other source.

Anyway, I think these details belong in the impl, not in the API.

Reply:
Changed so it's all `string_view` in the Impl